learning/step_by_step/scripting_continued.rst

```diff
@@ -29,9 +29,13 @@ frames per second (FPS) of the application:
 The delta parameter describes the time elapsed (in seconds, as
 floating point) since the previous call to "_process()".
 
-Fixed processing is similar, but only needed for synchronization with
-the physics engine. It is called a fixed amount of times per second
-(defined in the Project Settings, but 60 by default).
+Physics processing is similar, but should be used for all the processes that
```

Comment:
Reworked, closer to the [writing guidelines](docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html)
`Physics processing is similar, but it should be used for all the processes that must happen before each physics step. For example, to move a character.
It is always runs before a physics step and it is called at fixed time intervals, 60 times per second by default. Change the value in the Project Settings.`